pkg/kapp/clusterapply/waiting_changes.go

```diff
@@ -81,6 +90,10 @@ func (c *WaitingChanges) WaitForAny() ([]WaitingChange, error) {
 			desc := fmt.Sprintf("waiting on %s", change.Cluster.WaitDescription())
 			c.ui.Notify(descMsgs)
 
+			if err != nil && err.Error() == "resource wait timeout" {
```

Comment:
below is a check for err != nil. why dont we just use that instead of making a special case for this?

Reply:
Done. Changed the logic accordingly